trellis/examples/service_client/app.cpp

```diff
@@ -9,14 +9,12 @@ using namespace trellis::examples::proto;
 
 App::App(const Node& node, const Config& config)
     : client_{node.CreateServiceClient<AdditionService>()},
-      timer_{node.CreateTimer(config["examples"]["service"]["interval_ms"].as<unsigned>(), [this]() { Tick(); })},
+      timer_{node.CreateTimer(
+          config["examples"]["service"]["interval_ms"].as<unsigned>(), [this]() { Tick(); },
+          config["examples"]["service"]["initial_delay_ms"].as<unsigned>())},
```

Comment:
nice, was this param always there?

Reply:
timer supports an initial delay. the examples weren't using it until now.